trunk/SUAVE/Methods/Aerodynamics/Airfoil_Panel_Method/heads_method.py

```diff
@@ -0,0 +1,105 @@
+## @ingroup Methods-Aerodynamics-Airfoil_Panel_Method
+# heads_method.py 
+# Created:  Mar 2021, M. Clarke
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+import SUAVE 
+import numpy as np
+from scipy.interpolate import interp1d
+from scipy.integrate import odeint
+# ----------------------------------------------------------------------
+# heads_method.py 
+# ----------------------------------------------------------------------   
+## @ingroup Methods-Aerodynamics-Airfoil_Panel_Method
+def heads_method(del_0,theta_0, del_star_0, L, Re_L, x_i, Ve_i, dVe_i):
+    """ Computes the boundary layer characteristics in turbulen
+    flow pressure gradients
+
+    Assumptions:
+    None
+
+    Source:
+    None
+
+    Inputs: 
+
+    Outputs: 
+
+    Properties Used:
+    N/A
+    """        
+    n            = 100
```

Comment:
don't use fixed numbers

Reply:
I took it out of the file completely and its not part of the airfoil analysis function